protoc_plugin/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   fixnum: ^0.10.5
   path: ^1.0.0
-  protobuf: ^0.13.0
+  protobuf: ^0.14.0
```

Comment:
Let's wait a month before publishing a new breaking version
We could consider writing
0.14.0-dev,